packages/esm-patient-notes-app/src/notes/visit-notes-form.scss

```diff
@@ -70,7 +70,6 @@
   display: grid;
   grid-template-columns: 50% 10% 1fr;
   margin: 0.5rem 1rem;
-  width: 100%;
```

Comment:
missed this out when Removing side scrolling in allergies, start visit and visit notes forms